assets/src/components/StyledTextField.js

```diff
@@ -1,16 +1,21 @@
 import TextField from '@material-ui/core/TextField'
 import { withStyles } from '@material-ui/core/styles'
 
+const orange = '#ffae42'
+
 const StyledTextField = withStyles({
   root: {
     '& .MuiFormLabel-root.Mui-error': {
-      color: '#ffae42'
+      color: orange
```

Comment:
@justin0022, can you look into setting and utilizing the `warning` color in the theme palette? I think we can override the default `warning`, maybe with what we have in `WarningBanner` (`"#FFF3CD"`?). Maybe that's too light, but seems like we should somehow standardize uses of orange in the application.

Palette: https://material-ui.com/customization/palette/#palette-colors

Reply:
Maybe whatever you use for the `dark` part could be what you use in this component.

Reply:
Sure, will do!